lib/gradient/elixir_file_utils.ex

```diff
@@ -90,4 +118,13 @@ defmodule Gradient.ElixirFileUtils do
         []
     end
   end
+
+  defp ensure_module_loaded_into_gradualizer(binary) do
+    {path, _} = System.cmd("mktemp", [])
+    sanitized_path = path |> String.trim() |> Kernel.<>(".beam")
+
+    :file.write_file(to_charlist(sanitized_path), binary)
```

Comment:
```suggestion
    :ok = :file.write_file(to_charlist(sanitized_path), binary)
```

If it doesn't succeed, we won't get correct results, so it's better to fail fast rather than scratch your head if something's wrong later.

Reply:
Good point, added match to `:ok`